kafka/Dockerfile

```diff
@@ -4,7 +4,69 @@
 # Ignoring DL4006 globally because we inherit the SHELL from our base image
 # hadolint global ignore=DL3038,DL4006
 
-FROM stackable/image/java-base AS builder
+# Not tagging base image because it is built as part of the same process
+# hadolint ignore=DL3006
+FROM stackable/image/java-base as kafka-builder
+
+ARG PRODUCT
+ARG SCALA
+ARG OPA_AUTHORIZER
+ARG JMX_EXPORTER
+
+RUN microdnf install -y \
```

Comment:
```suggestion
RUN microdnf install \
```

Reply:
microdnf update missing before this

Reply:
Please have a look at e17a87f2c4e52956cbd3695292239e4614bd435f